src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+    rel="preconnect"
+    href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="moyo-logo" href="#home"></a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+            href="#apple"
+            class="nav__link is-active">Apple</a>
+          </li>
```

Comment:
- please add empty line for every `<li>` block, check the checklist
- please also align a tag code
<img width="1201" alt="Screenshot 2023-10-05 at 14 13 28" src="https://github.com/mate-academy/layout_moyo-header/assets/64746943/1e105554-9862-4319-abe2-6632fbb1387f">